scarb/src/core/publishing/source.rs

```diff
@@ -53,6 +58,11 @@ fn push_worktree_files(pkg: &Package, ret: &mut Vec<Utf8PathBuf>) -> Result<()>
                 return false;
             }
 
+            // Skip README and LICENSE files
+            if path == readme || path == license_file {
```

Comment:
Are `readme` and `license_file` absolute here?

Reply:
Yes, they are